socorro/cron/jobs/ftpscraper.py

```diff
@@ -290,9 +283,9 @@ def download(self, url):
         assert response.status_code == 200, response.status_code
         data = response.content
 
-        if is_caching:
-            with open(fn, 'w') as fp:
-                fp.write(data)
+        # All the things we're downloading should be text files
+        if not isinstance(data, six.text_type):
+            data = data.decode('utf-8')
```

Comment:
This fixes the problem in Python 3 where data is a bytes and the we go do string things on it and it gets very sad.